common/djangoapps/entitlements/models.py

```diff
@@ -301,3 +309,19 @@ def get_active_entitlements_for_user(cls, user):
             expired_at__isnull=False,
             enrollment_course_run=None
         ).select_related('user').select_related('enrollment_course_run')
+
+
+class CourseEntitlementSupportDetail(TimeStampedModel):
+    """
+    Table recording support interactions with an entitlement
+    """
+    entitlement = models.ForeignKey('entitlements.CourseEntitlement')
+    support_user = models.ForeignKey(settings.AUTH_USER_MODEL, null=True)
```

Comment:
Is there a reason we are allowing this field to be null?  Don't we always want to know the Support user?